.github/workflows/graph-pixi-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Regression Label
```

Comment:
This should be renamed to something like "Graph Pixi Tests". 
That will make it easier to tell if it is working or not. It currently has the same name as another workflow.

Below are the results right now. This workflow is the one with the `on: push` below it. It doesn't have any running jobs because of the syntax error (see comment below).
<img width="578" alt="image" src="https://github.com/user-attachments/assets/dd826431-6ab3-4557-9880-2d9648eb0394" />